### PR TITLE
EIP-15: Updating watcher version, documentation and dependent package.

### PR DESCRIPTION
**Pull request by Ruhanga**
Ticket: https://issues.openmrs.org/browse/EIP-15

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,8 +14,9 @@
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <maven.compiler.source>1.8</maven.compiler.source>
       <maven.compiler.target>1.8</maven.compiler.target>
-      <camel-version>3.3.0</camel-version>
+      <camelVersion>3.3.0</camelVersion>
       <testContainersVersion>1.15.2</testContainersVersion>
+      <eipVersion>1.0.0-SNAPSHOT</eipVersion>
```
Comment (rbuisson) [inline pom.xml:19]: Let's rename to `eipClientVersion`

**Review by rbuisson — CHANGES_REQUESTED** [review]

**Review by rbuisson — APPROVED** [review]

**Review by enyachoke — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -8,17 +8,19 @@
 
 Follow instructions here: https://github.com/openmrs/openmrs-distro-referenceapplication/tree/3.x
 
-#### Open port `3306` + enable the bin log:
-If running in Docker, you can do it by setting the `ports` and overriding the `command`:
+#### Enable mysql bin log
+IF using docker-compose, you can do it by exposing the `port` and overriding the `command` parameters for example;
```
Comment (rbuisson) [inline README.md:12]: If using Docker Compose, expose the `port` and override the `command` parameters in **docker-compose.yml**. For example:

File: README.md [hunk 3]
```diff
@@ -31,65 +33,9 @@ and follow the steps to "Install SENAITE"
...
-java -jar -Dspring.profiles.active=sender app/target/openmrs-eip-app-1.0-SNAPSHOT.jar
-
-```
-## 4. Add SENAITE routes to it
-
-```
-rsync -av routes/ ../openmrs-eip/distribution/springboot_setup/sender/routes/
-```
+#### Note
+Remember to provide an `application.properties` file defining the required runtime properties needed by the routes. Use this [template file](src/test/resources/application.properties). Uncomment the properties prefixed with `##` and set them appropriately.
```
Comment (rbuisson) [inline README.md:41]: Personally I tend to put file names in bold and pieces of code or variable names in code formatting.
I see that in EIP Client the "application.properties" file name is in bold. Let's be consistent and do that here too.

Unless there is an hyperlink, in which case it doesn't need to be bold.

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Ruhanga) [reply README.md:41]: Yes, 'template file' is simply a hyperlink to the file.

**Pull request merged**